docs/CONFIG-PROPERTIES.md

```diff
@@ -29,6 +29,7 @@
 | timer.port.testbetterinterval | timer in seconds | 600 | test a higher prio port config |
 | network.fallback.any.eth | "enabled" or "disabled" | disabled (enabled forcefully during onboarding if no network config) | if no connectivity try any Ethernet, WiFi, or LTE with DHCP client |
 | network.download.max.cost | 0-255 | 0 | [max port cost for download](DEVICE-CONNECTIVITY.md) to avoid e.g., LTE ports |
+| blob.download.max.retries | 1-10 | 5 | max download retries when image verification fails.|
```

Comment:
As far as I can see, we don't limit it to 10.

Reply:
We do limit it to 10 :)

Reply:
FWIW that limit in global.go where BlobDownloadMaxRetries is defined.